paper.md

```diff
@@ -51,9 +51,9 @@ authors:
 # - name: Michael Dayan  # 12 commits, issues: opened 0 participated in 0
 #   orcid: 
 #   affiliation:  # add full names if not yet listed, or indexes if already are
-# - name: Dae Houlihan  # 12 commits, issues: opened 0 participated in 0
-#   orcid: 
-#   affiliation:  # add full names if not yet listed, or indexes if already are
+ - name: Sean Dae Houlihan  # 12 commits, issues: opened 0 participated in 0 # First: Sean Dae; Last: Houlihan
+   orcid: 0000-0001-5003-9278
+   affiliation:  1, Department of Brain and Cognitive Sciences, Massachusetts Institute of Technology
```

Comment:
```suggestion
   affiliation:  2
```

and please add the `Department of Brain and Cognitive Sciences, Massachusetts Institute of Technology` as the affiliation index 2 on line 126 among affiliations